firefox-ios/firefox-ios-tests/Tests/XCUITests/AuthenticationTest.swift

```diff
@@ -21,16 +21,16 @@ class AuthenticationTest: BaseTestCase {
         waitForElementsToExist(
             [
                 app.alerts.buttons["Cancel"],
-                app.alerts.buttons["Sign in"]
+                app.alerts.buttons["Log in"]
```

Comment:
let's clarify this... Yesterday it was changed to Sign in, @mdotb-moz is also seeing Sign in not Log in..

Reply:
Well, on the last run the test failed, because of that reason, locally i could reproduced it, not sure what happen't since yesterday on this area:
https://storage.googleapis.com/mobile-reports/public/firefox-ios-M1/FullFunctionalTests/result_1189/build/reports/index.html

Reply:
Sign is is when you open the url on web. On mobile is login

Reply:
thank you